### DISCUSSION TITLE
screen wall

### DISCUSSION THREAD

**Discussion by jtdn608** [rustdesk/rustdesk#878] [Ideas]
Can the use of the internal network and the external network be isolated, and then the function of the screen wall can be added?

**Comment by rustdesk** [comment]
screen wall ?

**Reply by Storm937** [reply]
Operations personnel can monitor multiple computers in real time, neatly arranged on a single screen.
![20230328160645](https://user-images.githubusercontent.com/129156975/228170793-bc2bbb4a-6263-442f-8d7d-afa3a129fa05.png)

**Reply by rustdesk** [reply]
which industry？

**Reply by Storm937** [reply]
Government, enterprise, school real-time monitoring computer, studio mass machine monitoring.

**Comment by Storm937** [comment]
We are the only two people in the entire community who have this need. Is there any other way to achieve it?

**Reply by OnSive** [reply]
You can hope that #3967 is implemented in the near future, then you or anyone else could easily write their own plugins.
